jsapp/kobo/directives/KobocatFormPublisher.Directive.js

```diff
@@ -15,7 +15,7 @@ kobo.directive ('kobocatFormPublisher', ['$api', '$miscUtils', '$routeTo', funct
                     $('button.save-button').removeClass('save-button--deploying');
                     scope.close();
                     $miscUtils.alert('Survey Publishing succeeded');
-                    $routeTo.external(results.published_form_url);
+                    // $routeTo.external(results.published_form_url);

```

Comment:
The commented line triggers redirect to ona's form page with the same string as shown below:
![screen shot 2015-05-04 at 2 00 44 pm](https://cloud.githubusercontent.com/assets/1128227/7452392/b943a11e-f265-11e4-9723-dcfd2875bd8a.png)

I thought it was best to comment it out because currently on map your world, once a form has been published, it redirects to the `maps` tab which has a listing of your published forms. The ideal scenario would be to redirect to the maps tab when a survey has been created but if we plan to use this on zebra too, we would want this either to redirect to the form listing page in zebra or may be just have a link to the submitted form on the pop-up that has 'Successfully submitted'.